src/js/media-controller.js

```diff
@@ -880,6 +936,28 @@ const monitorForMediaStateReceivers = (
   return unsubscribe;
 };
 
+let testMediaEl;
+export const getTestMediaEl = (nodeName = "video") => {
+  if (testMediaEl) return testMediaEl;
+  if (typeof window !== "undefined") {
```

Comment:
yeah... we need to figure out how we want to handle "SSR meets our polyfills" cases. From an unpolyfilled use case for envs like JSDOM, checking for `window` is common, but since we polyfill, may need to tweak this. to handle our very specific env.